src/blog/2023/07/images-in-node-red-dashboards.md

```diff
@@ -24,6 +24,12 @@ When designing a dashboard, images allow you to significantly enrich your conten
 
 - displaying physical tools which should be used to resolve a problem.
 
+## Easily Add Images to Node-RED Dashboards with FlowFuse’s Static Asset Service
+
+The FlowFuse static assets service offers a user-friendly and efficient way to manage your static assets within Node-RED. This service simplifies asset management by allowing users to handle everything within the same instance through a convenient interface. For detailed instructions on using the static assets service, refer to the [FlowFuse Assets Service Documentation](https://flowfuse.com/docs/user/static-asset-service/).
```

Comment:
@sumitshinde-84 Can we describe step by step how to get a logo or so on a Dashboard?